dashboard/app/controllers/home_controller.rb

```diff
@@ -86,31 +86,24 @@ def init_homepage
       @sections = current_user.sections.map(&:summarize)
       @student_sections = current_user.sections_as_student.map(&:summarize)
 
-      # Students will receive a @student_top_course for their primary script,
-      # so we don't want to include that script (if it exists) in the regular
-      # lists of recent scripts.
-      exclude_primary_script = current_user.student?
+      # Students and teachers will receive a @top_course for their primary script, so we don't want to include that script (if it exists) in the regular lists of recent scripts.
+      exclude_primary_script = current_user
```

Comment:
we already know current_user be truthy here (see line 81).

also, nit: i'd prefer to see line 89 wrapped